core/api.txt

```diff
@@ -581,6 +581,10 @@ ion-item,css-prop,--border-color
 ion-item,css-prop,--border-radius
 ion-item,css-prop,--border-style
 ion-item,css-prop,--border-width
+ion-item,css-prop,--bottom-background
+ion-item,css-prop,--bottom-padding-bottom
+ion-item,css-prop,--bottom-padding-start
+ion-item,css-prop,--bottom-padding-top
```

Comment:
Would it be possible to only fix the issue in https://github.com/ionic-team/ionic-framework/issues/23892 for now, and create a ticket to add the CSS vars in a 6.x release? I think it is more important to have consistency with Ionic 5 behavior than full customization.